var/spack/repos/builtin/packages/mvapich2-gdr/package.py

```diff
@@ -23,6 +23,7 @@ class Mvapich2Gdr(AutotoolsPackage):
 
     maintainers = ["ndcontini", "natshineman", "harisubramoni"]
 
+    version('2.3.7', sha256='7bf748ed3750aa607382fc96229e256d888824aed758ce364b1ed9429da4779e')
```

Comment:
Need to replace single with double quotes here to pass style/black checks.

Reply:
FYI. Cannot access the site to confirm the sha256 (without registering).

Reply:
Unfortunately even after going through that page the sha won't really be verifiable. MVAPICH2-GDR, along with MVAPICH2-X are closed source packages that are only available through our buildcache so the tarball is not actually posted anywhere publicly. We mostly update these for our internal use, but also so that users can see when we've added newer versions of the package. 

We are in the process of improving our Spack deployments, would there be a better way to handle this? Or should simply remove the versions altogether?

Reply:
> Unfortunately even after going through that page the sha won't really be verifiable. MVAPICH2-GDR, along with MVAPICH2-X are closed source packages that are only available through our buildcache so the tarball is not actually posted anywhere publicly. We mostly update these for our internal use, but also so that users can see when we've added newer versions of the package.
> 
> We are in the process of improving our Spack deployments, would there be a better way to handle this? Or should simply remove the versions altogether?

@tgamblin @alalazo Maybe you have opinions on conventions for packages like this?